src/screens/Asset/ReceiveModal.js

```diff
@@ -232,3 +154,60 @@ const combinedMapStateToProps = (state: RootReducerState): $Shape<StateProps> =>
 });
 
 export default (connect(combinedMapStateToProps)(ReceiveModal): AbstractComponent<OwnProps>);
+
+const ContentWrapper = styled(SafeAreaView)`
+  padding: 0 ${spacing.layoutSides}px 30px;
+  align-items: center;
+`;
+
+const QRCodeWrapper = styled.View`
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  margin: ${spacing.largePlus}px;
+`;
+
+const WalletAddress = styled(BaseText)`
+  ${fontStyles.regular};
+  color: ${({ theme }) => theme.colors.basic030};
+  margin: ${spacing.mediumLarge}px;
+  text-align: center;
+`;
+
+const WalletAddressWithCopy = styled(TextWithCopy)`
+  ${fontStyles.regular};
+  color: ${({ theme }) => theme.colors.basic030};
+  text-align: center;
+`;
+
+const CopyButton = styled.View`
+  width: 100%;
+  justify-content: space-between;
+  margin-top: ${spacing.largePlus}px;
+  margin-bottom: ${spacing.small}px;
+`;
+
+const ShareButton = styled.View`
+  width: 100%;
+  justify-content: space-between;
+  margin-bottom: ${spacing.medium}px;
+`;
+
+const InfoView = styled.View`
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const ImageWrapper = styled.View`
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+`;
+
+const WarningText = styled(BaseText)`
```

Comment:
Pls modern/Text component instead of BaseText for new work

Reply:
Done